### PR TITLE
Add CENDL-3.2 Library

### PR DESCRIPTION
**Pull request by AI-Pranto**
For zip file download, I've used [CNDC](http://www.nuclear.csdb.cn/cendl32.htm) web-link because OECD NEA didn't include it in their data bank but it's available from [IAEA-NDS](https://www-nds.iaea.org/public/download-endf/).

### CODE REVIEW

File: generate_cendl.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
                     default='latest', help="Output HDF5 versioning. Use "
                     "'earliest' for backwards compatibility or 'latest' for "
                     "performance")
-parser.add_argument('-r', '--release', choices=['3.1'],
+parser.add_argument('-r', '--release', choices=['3.1', '3.2'],
                     default='3.1', help="The nuclear data library release "
```
Comment (Shimwell) [inline generate_cendl.py:43]:
```suggestion
                    default='3.2', help="The nuclear data library release "
```

Comment (Shimwell) [reply generate_cendl.py:43]: Perhaps the newest should be the default as in the other scripts

**Comment by Shimwell** [comment]
Sorry about committing that ```<br>``` directly, I got a bit carried away there. Nice addition to the table :+1:

**Comment by AI-Pranto** [comment]
Thanks @Shimwell for removing the `<p>` tag.

**Comment by Shimwell** [comment]
Sorry this took me so long, these scripts involving NJOY conversions can take a while to run.

Happy to report that everything worked over here, the script downloaded and processed all the data nicely :tada:

**Review by Shimwell — APPROVED** [review]

**Comment by paulromano** [comment]
Thanks @AI-Pranto for this contribution. Sorry it fell off my radar before!

**Pull request merged**